operations/alloy-mixin/alerts/clustering.libsonnet

```diff
@@ -9,7 +9,7 @@ local alert = import './utils/alert.jsonnet';
         alert.newRule(
           'ClusterNotConverging',
           if enableK8sCluster then 
-            'stddev by (cluster, namespace, job) (sum without (state) (cluster_node_peers)) != 0' 
+            'stddev by (cluster, namespace, job, cluster_name) (sum without (state) (cluster_node_peers)) != 0' 
```

Comment:
We should name this new label as `alloy_cluster`, to clearly differentiate from the k8s `cluster` label. This is also the standard we have been using in other integrations, like `kafka_cluster`, `mongodb_cluster`, etc.

Reply:
the problem is that the cluster metrics are created by ckit library which is external to Alloy (https://github.com/grafana/ckit)

Reply:
Ok, got it. Lets move on.